backend/app/api/game.py

```diff
@@ -157,10 +157,25 @@ def get_tile():
 
     return tile
 
-
+"""function to filter chat messages"""
 def filter_message(data: Dict[str, str]) -> Dict[str, str]:
-    """function to filter chat messages"""
 
+    bannedWords = ["shit","fuck","crap","bitch"]
+    censoredSentence = []
+    " ".join(censoredSentence)
+    
+    data = {
+        type : message,
+        Message : censoredSentence,
+        fromUser : username,
+        }
+    
+    for word in Message.split():
```

Comment:
there is no variable called `Message` in this scope. You should be iterating over `data['message']`.